lib/ansible/modules/windows/win_psexec.py

```diff
@@ -65,6 +65,11 @@
     - Run the program so that it interacts with the desktop on the remote system.
     type: bool
     default: 'no'
+  session:
+    description:
+    - In conjunction with interactive, specify a session for psexec to attach to. Has no effect if interactive is 'no'.
```

Comment:
We prefer if the first entry of the description is the summary of what the parameter is for.
Subsequent entries can clarify how it is used. So I would do:
```
  session:
    description:
    - Specifies the session ID to use.
    - This parameter works in conjunction with parameter C(interactive).
    - It has no effect when C(interactive) is set to C(no).
    type: int
    version_added: '2.7'
```